src/Site.js

```diff
@@ -691,7 +688,6 @@ class Site {
     const filePathArray = Array.isArray(filePaths) ? filePaths : [filePaths];
     const uniquePaths = _.uniq(filePathArray);
     logger.info('Rebuilding affected source files');
-    MarkBind.resetVariables();
```

Comment:
Is there a reason why we no longer need to reset the variables? What if a variable has been deleted? Would the old value associated with the variable also be cleared when rebuilding the page?

Reply:
To add, the process before essentially involved 2x extraction and 2x rendering, storing the intermediate extracted variables in `Parser`'s static variables.
The process is now pure since its just one step, so we have no need for that.